api/server/router/local/image.go

```diff
@@ -433,6 +425,39 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	return nil
 }
 
+// repoAndTag is a helper struct for holding the parsed repositories and tags of
+// the input "t" argument.
+type repoAndTag struct {
+	repo, tag string
+}
+
+// parseRepoAndTags parses the raw "t" argument received from the client
+// to a slice of repoAndTag.
+// It also validates each repoName and tag.
+func parseRepoAndTags(names []string) ([]repoAndTag, error) {
```

Comment:
@endophage thanks for reviewing this.
I think remove the duplicated entries should be okay. No need to raise the error. 
I noticed that if I provide duplicated `--label` or `--env` flags to `docker run` there are no errors either.
Anyway, I'll push another commit to raise the error for discuss.